### ISSUE TITLE
PLEASE UPDATE INSTALLATION GUIDE START COMMAND

### ISSUE THREAD

**Issue by samsong** [getumbrel/umbrel#700]
sudo /start does not work on ubuntu linux.

**error:**
touch: cannot touch '/home/user/umbrel/../statuses/configured': No such file or directory

MULTIPLE other users had this error (check telegram chat)

**You need this command**
cd && mkdir statuses -p && cp umbrel/statuses/* statuses/ && cd umbrel && sudo ./scripts/start

Please update your installation manual on github.

**Comment by louneskmt** [comment]
Hey! This is not the issue and won't solve it. I will take a look tomorrow.

**Comment by samsong** [comment]
Ok well by running this command instead i was able to fully start umbrel.
So hopefully this helps you to find the real issue & fix it.
Because I saw in telegram chat history for umbrel this was mention by multiple users months ago even so seems like something consistent.

**Comment by blckbx** [comment]
Is it: `UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."` ? Why are the dots `/..` needed?

**Comment by AaronDewes** [comment]
${BASH_SOURCE[0]} is /home/ubuntu/scripts/configure
$(dirname $(readlink -f "${BASH_SOURCE[0]}")) is /home/ubuntu/umbrel/scripts
$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.. is /home/ubuntu/umbrel/scripts/.. = /home/ubuntu/umbrel

**Comment by Ruppsn** [comment]
> sudo /start does not work on ubuntu linux.

Why are you using `sudo /start`?

The Documentation i have in front of me states: `sudo ./scripts/start` Or did you mean sudo ./scripts/start ?

Its not easy to help if you are giving wrong informations but i will try : /

Can you give me the output of `bash --version` ? Are you using somthing different than Bash? CSH? ZSH? I had some Problems with BASH_SOURCE in my scripts wenn i wasn't using another host withoout bash.

**Comment by samsong** [comment]
Yes I meant the full command you said (i edited my original post)

bash --version
"GNU Bash 5.0.17 release x86_64-pc-linux-gnu"

This is fresh install of Lubuntu linux (ubuntu with LXQT desktop nothing much different)

Umbrel directory is located in: /home/user(myusername)/umbrel/

Let me know any other commands you want me to run or information to provide.

**Comment by Darth-Coin** [comment]
Is this related with Ubuntu 18.04? Seems that in 20.04/21.04 this doesn't happen.
I use Debian 10 for my Umbrel and never have this problem.

**Comment by ildella** [comment]
this happens in Ubuntu 21.04

**Comment by AaronDewes** [comment]
> this happens in Ubuntu 21.04

Can you please put these three commands after the first line (but in a new line) of scripts/start so we can get a bit of information?


```
echo ${BASH_SOURCE[0]}
echo $(dirname $(readlink -f "${BASH_SOURCE[0]}"))
echo $(dirname $(readlink -f "${BASH_SOURCE[0]}"))/..
```

**Comment by algotechio** [comment]
I had the same problem on 20.04.  To reproduce:

1. I ran `curl -L https://github.com/getumbrel/umbrel/archive/v0.3.10.tar.gz | tar -xz --strip-components=1` in home directory
2. When I realized that the unzip happens without creating a directory, I removed all files and created `umbrel` directory and again ran the command in umbrel directory
3. HOWEVER, the file `~/.umbrel` was left behind in the home directory
4. Then the code (line 39 in `scripts/configure`), sets the STATUS_DIR to `~/statuses` instead of the directory holding the source code (in my case `~/umbrel/statuses`
```
# Make sure we use the status dir from the real Umbrel installation if this is
# an OTA update.
STATUS_DIR="${UMBREL_ROOT}/statuses"
if [[ -f "${UMBREL_ROOT}/../.umbrel" ]]; then
  STATUS_DIR="${UMBREL_ROOT}/../statuses"
fi
```

I think to prevent this error, I would modify the `README.md` and remove the `--strip-components=1` from the command:

```
curl -L https://github.com/getumbrel/umbrel/archive/v0.3.10.tar.gz | tar -xz --strip-components=1
```
as this can be easily overlooked and then the users just dump the source code files in the current working directory. 

Can I help with a pull request for this change?

**Comment by AaronDewes** [comment]
> as this can be easily overlooked and then the users just dump the source code files in the current working directory.
Can I help with a pull request for this change?

I think it should be kept as it is now, because there is a note before the command:

> Run this in an **empty** directory where you want to install Umbrel.


But it's best to let @lukechilds decide

**Comment by mrWiga** [comment]
> I had the same problem on 20.04. To reproduce:
> 
> 1. I ran `curl -L https://github.com/getumbrel/umbrel/archive/v0.3.10.tar.gz | tar -xz --strip-components=1` in home directory
> 2. When I realized that the unzip happens without creating a directory, I removed all files and created `umbrel` directory and again ran the command in umbrel directory
> 3. HOWEVER, the file `~/.umbrel` was left behind in the home directory
> 4. Then the code (line 39 in `scripts/configure`), sets the STATUS_DIR to `~/statuses` instead of the directory holding the source code (in my case `~/umbrel/statuses`
> 
> ```
> # Make sure we use the status dir from the real Umbrel installation if this is
> # an OTA update.
> STATUS_DIR="${UMBREL_ROOT}/statuses"
> if [[ -f "${UMBREL_ROOT}/../.umbrel" ]]; then
>   STATUS_DIR="${UMBREL_ROOT}/../statuses"
> fi
> ```
> 
> I think to prevent this error, I would modify the `README.md` and remove the `--strip-components=1` from the command:
> 
> ```
> curl -L https://github.com/getumbrel/umbrel/archive/v0.3.10.tar.gz | tar -xz --strip-components=1
> ```
> 
> as this can be easily overlooked and then the users just dump the source code files in the current working directory.
> 
> Can I help with a pull request for this change?

Yep, exact same problem here. Moving .umbrel to the folder fixed for me. thanks for me

**Comment by onepabz** [comment]
still happens today in a freshly installed ubuntu 20.04

**Comment by PatMulligan** [comment]
ran into this same issue...

**Comment by docter-tirrud** [comment]
And today I did, too. Tjhanks to op for posting.
